From f79d948f42d3da2fc7360917d237aece14b7523d Mon Sep 17 00:00:00 2001
From: dimkaps <150019158+dimkaps@users.noreply.github.com>
Date: Fri, 17 Oct 2025 18:21:51 +0100
Subject: [PATCH] Enable TLV320AIC3X driver (#35)

This commits enables the TLV320AIC3X and TLV320AIC3X-I2S kernel drivers.
These are used to drive the TLV320AIC3101 chip.
Note that even though the TLV320AIC31XX driver seems to
have compatible devices which are closer named to this chip (ti,tlv320aic31xx),
the correct driver is the TLV320AIC3X because it contains the correct registers
for the chip and controls (HPOUT, HPCOM, MICs etc).

Datasheet for reference:
https://www.ti.com/lit/ds/symlink/tlv320aic3101.pdf
---
 arch/arm64/configs/vendor/kalama_le_consolidate.config | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/vendor/kalama_le_consolidate.config b/arch/arm64/configs/vendor/kalama_le_consolidate.config
index a06e7d941..125a680be 100644
--- a/arch/arm64/configs/vendor/kalama_le_consolidate.config
+++ b/arch/arm64/configs/vendor/kalama_le_consolidate.config
@@ -38,3 +38,5 @@ CONFIG_RTW88=m
 CONFIG_RTW88_PCI=m
 CONFIG_RTW88_8822CE=m
 CONFIG_SENSORS_SSC=m
+CONFIG_SND_SOC_TLV320AIC3X_I2C=m
+CONFIG_SND_SOC_TLV320AIC3X=m
-- 
2.41.0

